### PR TITLE
Empty example table entries with converter to None documentation

### LINKED ISSUE #551: New pytest bdd 6.0.1 doesnt take null values from the Examples within the scenario outline
Steps to reproduce

1. Within a Pytest feature file try to have a scenario outline
2. Within example (just take example for a login page) as discussed
3.Scenario Outline: Login functionality with Negative password  and valid email
        Given User opens login in page
        When the User enters <invalid_password> into password and valid credential on the other textbox
        Then verify wrong_password will display a <warning_message> under the textbox
        Examples:
        |  invalid_password     |    warning_message                       |
        |                                    |    Password cannot be empty        |
        
        Step definition for the above scenario :
        def test_login_invalidPassword_Cases():
        pass

@when(parsers.parse('the User enters {invalid_password} into {textbox} and valid credential on the other textbox'))
def login_with_negative_password(textbox, invalid_password):
    Login_functions.fn_InvalidLogin(textbox, invalid_password)
        
  4. In the run time of the test ,the null value is getting as " " (double qoutes)

### PR DESCRIPTION
**Pull request by jsa34**
Add documentation for parser converter for empty strings in example tables.

Resolves #551

### CODE REVIEW

File: tests/feature/test_outline_empty_values.py [hunk 1]
```diff
@@ -62,3 +62,65 @@ def test_outline():
     result = pytester.runpytest("-s")
     result.assert_outcomes(passed=1)
     assert collect_dumped_objects(result) == ["#", "", ""]
+
+
+def test_scenario_with_empty_example_values_none_transformer(pytester):
```
Comment (youtux) [inline tests/feature/test_outline_empty_values.py:67]: please add a docstring explaining what's the issue we are covering here

Comment (jsa34) [reply tests/feature/test_outline_empty_values.py:67]: Good point - I'll also add an example in the README as I realised this is probably the helpful place for people wanting to know how to do it

**Comment by jsa34** [comment]
@youtux Could you review and see if this is clear/correct/helpful?

**Review by youtux — APPROVED** [review]
do we really need a test for this? isn't this tested somewhere else already? Maybe not this very example, but the fact that converters... basically work.

**Comment by jsa34** [comment]
Good point.  I'll just add to the docs

**Review by youtux — APPROVED** [review]

**Pull request merged**